frontend/src/entry/settings/ManualEntryAccess.tsx

```diff
@@ -0,0 +1,67 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import { Accordion, AccordionDetails, AccordionSummary, Button, Stack, TextField, Typography } from '@mui/material'
+import { getUserInformation } from 'actions/user'
+import { useState } from 'react'
+import MessageAlert from 'src/MessageAlert'
+import { CollaboratorEntry, EntityKind } from 'types/types'
+import { getErrorMessage } from 'utils/fetcher'
+
+interface ManualEntryAccessProps {
+  accessList: CollaboratorEntry[]
```

Comment:
This component doesn't have to care about the access list. It can probably have a single prop `onAddEntityManually: (entityName: string) => void`. We can let the main `EntryAccessInput` component manage the access list logic, though I also think the checks we're doing within `handleAddEntityManuallyOnClick` can and should be handled by the backend.